source/applications/advanced/hand_eye_calibration/pose_conversions.py

```diff
@@ -266,7 +264,7 @@ def get_transformation_matrix_from_yaml(path: Path):
         4x4 Transformation Matrix
 
```

Comment:
I see you use type-annotations ("path: Path") some times, but not always. Do you have a convention?

In this particular case maybe not a good idea, because the function will also work with str (and that's maybe a feature).

Reply:
We don't have a convention. I've removed it from here. @torbsorb is the one who uses those sometimes.